pkg/rhcos/builds.go

```diff
@@ -22,6 +23,13 @@ var (
 	baseURL = "https://releases-rhcos.svc.ci.openshift.org/storage/releases"
 )
 
+func init() {
+	if or, ok := os.LookupEnv("OPENSHIFT_INSTALL_COREOS_IMAGE_URL"); ok && or != "" {
```

Comment:
why COREOS and not RHCOS ?

Reply:
I guess I'd vaguely thought that we want to enable e.g. pointing the installer at a custom Fedora CoreOS build with kubelet for testing.